test/services/account/moodle/moodleMockServer.js

```diff
@@ -11,27 +11,38 @@ module.exports = function MoodleMockServer({
 }) {
 	const findFreePort = port ? Promise.resolve(port) : freeport();
 
-	return findFreePort.then((freePort) => new Promise((resolve) => {
-		const mockMoodle = express();
-		mockMoodle.use(bodyParser.json()); // for parsing application/json
-		mockMoodle.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
-		mockMoodle.responseToken = responseToken;
-		mockMoodle.port = freePort;
-		mockMoodle.url = `http://localhost:${mockMoodle.port}`;
-		mockMoodle.post('/login/token.php', (req, res) => {
-			if (!acceptServices.includes(req.body.service)) {
-				res.send(`{"error": "Web service ${res.body.service} is not available (it doesn't exist or might be disabled)"}`);
-				return;
-			}
-			if (acceptUsers.find((user) => user.username === req.body.username && user.password === req.body.password)) {
-				res.json({ token: responseToken });
-			} else {
-				res.send('{"error": "Invalid login, please try again"}'); // HTTP response code is 200 in any case
-			}
-		});
+	return findFreePort.then(
+		(freePort) =>
+			new Promise((resolve) => {
+				const mockMoodle = express();
+				mockMoodle.use(bodyParser.json()); // for parsing application/json
+				mockMoodle.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
+				mockMoodle.responseToken = responseToken;
+				mockMoodle.port = freePort;
+				mockMoodle.url = `http://localhost:${mockMoodle.port}`;
+				mockMoodle.post('/login/token.php', (req, res) => {
+					if (!acceptServices.includes(req.body.service)) {
+						res.send(
+							`{"error": "Web service ${res.body.service} is not available (it doesn't exist or might be disabled)"}`
+						);
+						return;
+					}
+					if (
+						acceptUsers.find(
+							(user) =>
+								user.username === req.body.username &&
+								user.password === req.body.password
+						)
+					) {
+						res.json({ token: responseToken });
+					} else {
+						res.send('{"error": "Invalid login, please try again"}'); // HTTP response code is 200 in any case
```

Comment:
Codacy found an issue: [This line has a length of 122. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)